CMakeLists.txt

```diff
@@ -248,13 +248,13 @@ IF (PLUS_USE_OPTITRACK)
   IF(NOT BUILD_ARCHITECTURE MATCHES "x64")
     SET(MOTIVE_VERSION_DEFAULT "1.10.3")
   ELSE()
-    SET(MOTIVE_VERSION_DEFAULT "2.1.2")
+    SET(MOTIVE_VERSION_DEFAULT "2.2.0")
   ENDIF()
   SET(MOTIVE_VERSION ${MOTIVE_VERSION_DEFAULT} CACHE STRING "Choose the motive API version.")
   IF (TEMP_OS_ARCH MATCHES "x86" AND MOTIVE_VERSION VERSION_GREATER "2.1.0")
     MESSAGE(FATAL_ERROR "MOTIVE_VERSION cannot be greater than 2.1.0 for 32-bit builds")
   ENDIF()
-  set_property(CACHE MOTIVE_VERSION PROPERTY STRINGS "1.10.3" "2.1.1")
+  set_property(CACHE MOTIVE_VERSION PROPERTY STRINGS "1.10.3" "2.2.0")

```

Comment:
Should you leave in 2.1.1 and add 2.2.0 as another option?

Reply:
We could, I took it out because there are other  versions from before 2.11 in PLTools which aren't included (but would still be supported by the source code).

We should either add back in these older versions, or decide on only supporting the latest version of Motive 2.x. Since this makes testing easier my vote would be to only include 2.2 and ask users to upgrade (as was done on previous releases).